index.js

```diff
@@ -225,6 +229,41 @@ async function main (argv) {
   }
   process.stdout.write('OK')
 
+  //
+  //  Initialize tsconfig.json when react_ts 
+  //
+  if (templateName === 'react_ts') {
+    try {
+      process.stdout.write('\nCreating tsconfig....')
```

Comment:
Can you put all line feeds at the end of strings instead of the start?

Reply:
This is how it's look now, if I'll put all line feeds at the end it'll break layout 
``` 
Initializing npm package...Ok.
Installing developer dependencies...Ok.
Installing '@socketsupply/socket' locally (ssc not in PATH)
Installing dependencies...running pre req script at node_modules\@socketsupply\socket-win32-x64\bin\install-pre-reqs.js
Using toolchain: vsbuild
# C:\WINDOWS\System32\vcruntime140_1.dll found.
Git found at, changing path to: git.exe
# Found git.exe
Calling vcvars64.bat
Please run in future dev sessions: 
(Or just run cd C:\socketsupply\create-socket-app-react-ts\node_modules\@socketsupply\socket-win32-x64\ps1; .\bin\install.ps1)
$env:PATH='C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\Llvm\x64\bin'+$env:PATH
Ok.
Adding package scripts...Ok.
Creating socket files...Ok. 
Creating tsconfig...Ok.
Setting up TS configuration...Ok.   
Updating project configuration...Ok.
Copying project boilerplate...Ok.   

Type 'npm start' to launch the app
```

Reply:
Hm, is there an issue with the code that creates this line?
`Creating socket files...Ok.`
Can you try to fix that? If not, you can just start your output with a single `\n` to get things working normally.

Reply:
I had to removed trailing space right after the comment.  It's all fixed now.